pkg/provisioningv2/rke2/planner/etcdrestore.go

```diff
@@ -172,7 +172,7 @@ func (p *Planner) runEtcdRestoreControlPlaneEtcdServiceStop(controlPlane *rkev1.
 			return err
 		}
 	}
-	servers := collect(clusterPlan, anyRole)
+	servers := collect(clusterPlan, anyRoleWithoutWindows)
```

Comment:
nit: for now could this just be `notWindows`?

Reply:
I already created a notWindows function and this one is more descriptive as its the same as the anyRole but without Windows.